components/BottomSheet/src/MDCBottomSheetPresentationController.h

```diff
@@ -76,6 +76,13 @@
  */
 @property(nonatomic, assign) CGFloat preferredSheetHeight;
 
+/**
+ Customize the color of the background scrim.
+
+ Defaults to a semi-transparent Black.
```

Comment:
can you please provide the exact color value

Reply:
Just a thought, but keeping it vague would allow us some latitude in the future if we need to shift the transparency by a few percent.  What do you think, @yarneo?

Reply:
I copied the comment [from MDCDialogPresentationController](https://github.com/material-components/material-components-ios/blob/develop/components/Dialogs/src/MDCDialogPresentationController.h#L66) and I kind of agree that specificity would be likely to get stale.

I'll update the PR to add the properties to the controller as well.

Reply:
Added. I also put a note in the test about a peculiarity/gotcha in the propagation of that new property.  Note that this is *not* how it's implemented for Dialogs (the transition controller isn't used at all) and that implementation doesn't have the same gotcha.  For Bottom Sheet, however, this matches the other properties that find their way from the sheet controller to the presentation controller, so it fits better with existing code.

I'm happy to change this to match Dialogs, if that's preferable, or leave it for cleanup later.  Either way, it will be in a separate PR.